src/backend/vulkan/src/lib.rs

```diff
@@ -424,6 +429,13 @@ impl hal::Instance<Backend> for Instance {
             })?
         };
 
+        let get_device_physical_properties_found = extensions.iter().find(|&&ext| ext == *KHR_GET_PHYSICAL_DEVICE_PROPERTIES2);
+        let get_physical_device_properties = get_device_physical_properties_found.map(|_| {
```

Comment:
nit: any reason not to write this down as a single expression?`get_device_physical_properties_found` isn't used anywhere else

Reply:
Just a artifact of how I developed the code, will combine.